pkg/snapshot/controllers/snapshotschedule.go

```diff
@@ -169,24 +185,25 @@ func (s *SnapshotScheduleController) setDefaults(snapshotSchedule *stork_api.Vol
 	}
 }
 
-func getVolumeSnapshotStatus(name string, namespace string) (snapv1.VolumeSnapshotConditionType, error) {
+// getVolumeSnapshotStatus returns the status of the volumesnapshot passed in the parameter.
+func getVolumeSnapshotStatus(name string, namespace string) (snapv1.VolumeSnapshotConditionType, string, error) {
 	snapshot, err := k8sextops.Instance().GetSnapshot(name, namespace)
 	if err != nil {
-		return snapv1.VolumeSnapshotConditionError, err
+		return snapv1.VolumeSnapshotConditionError, "", err
```

Comment:
why are we not sending the error message here?

Reply:
The second string implies snapshot error. In this case, we haven't got the snapshot object yet hence not sending the error of snapshot.